### ISSUE TITLE
Redirect to the project page after login

### ISSUE THREAD

**Issue by abitrolly** [fedora-infra/anitya#913]
When I browse anonymously at https://release-monitoring.org/project/13305/ and  click `login` and `Fedora`, then after login I am redirected to https://release-monitoring.org/ and I wanted to stay on the same page.

Not sure it is easy to do. Need some place to store path to get back to across page reloads.

**Comment by Zlopez** [comment]
It's now working for redirection when you are asked by the Anitya to login. But I could look at this in the future.

**Comment by abitrolly** [comment]
You mean if I press the Edit button on the same page? Where does it store the return page address?

**Comment by Zlopez** [comment]
Yes, it stored to `next_url` in session cookie, which is taken from `next` field in request. When user gets redirected to `index` after authentication it's redirected automatically to the value in this field.

**Comment by abitrolly** [comment]
Didn't work for me. Started on https://release-monitoring.org/project/10350/ and clicked Edit. Sent to https://release-monitoring.org/login and clicked Fedora to login. Ended on https://release-monitoring.org/ after login.

**Comment by Zlopez** [comment]
We want to migrate to another authentication library in the future, so the login process will change and hopefully this will be solved as well.

I did tried your use case in Chromium based browser and the redirection worked for me. What browser are you using?

**Comment by abitrolly** [comment]
Firefox 100.

**Comment by abitrolly** [comment]
Well, it works the second time. Even from anonymous tab. Not sure what happened. Maybe the cookie got expired the first time. Very strange.

**Comment by abitrolly** [comment]
Ok. Now it didn't work, because this time I believe I hit login button in the top menu. I didn't expect page URL to change, and send me to the main.

So it is not completely fixed. To validate.

1. Open https://release-monitoring.org/project/132276/ and **hit Edit button**. The URL is changed to https://release-monitoring.org/login/?next=%2Fproject%2F132276%2Fedit Login with Fedora. Get redirected back to project page.This part works

2. Open https://release-monitoring.org/project/132276/ and **hit Login button**. The URL is change to https://release-monitoring.org/login Login with Fedora. Get redirected to main page. This part doesn't work.
